shared/src/models/radiogram/missing-transfer-connection-radiogram.ts

```diff
@@ -0,0 +1,48 @@
+import { IsBoolean, IsUUID, ValidateNested } from 'class-validator';
+import { UUID } from '../../utils';
+import { IsValue } from '../../utils/validators';
+import { IsRadiogramStatus } from '../../utils/validators/is-radiogram-status';
+import { getCreate } from '../utils';
+import type { Radiogram } from './radiogram';
+import { ExerciseRadiogramStatus } from './status/exercise-radiogram-status';
+
+export class MissingTransferConnectionRadiogram implements Radiogram {
+    @IsUUID()
+    readonly id: UUID;
+
+    @IsValue('missingTransferConnectionRadiogram')
+    readonly type = 'missingTransferConnectionRadiogram';
+
+    @IsUUID()
+    readonly simulatedRegionId: UUID;
+
+    /**
+     * @deprecated use the helpers from {@link radiogram-helpers.ts}
+     * or {@link radiogram-helpers-mutable.ts} instead
+     */
+    @IsRadiogramStatus()
+    @ValidateNested()
+    readonly status: ExerciseRadiogramStatus;
+
+    @IsBoolean()
+    readonly informationAvailable: boolean = false;
+
+    @IsUUID()
+    readonly targetTransferPointID: UUID;
+
+    /**
+     * @deprecated Use {@link create} instead
+     */
+    constructor(
+        id: UUID,
+        simulatedRegionId: UUID,
+        status: ExerciseRadiogramStatus
+    ) {
+        this.id = id;
+        this.simulatedRegionId = simulatedRegionId;
+        this.status = status;
+        this.targetTransferPointID = '';
```

Comment:
This should be a parameter of the constructor. `informationAvailable` should also be `true` by default, because this radiogram is created with all required information.

Reply:
Done